### PR TITLE
Custom airport development tools

### PR DESCRIPTION
**Pull request by ckwng**
These are a set of development tools written in Python 3 (requires Python 3.9). Currently what it can do is:

-Search for airport files under a 'source' folder, optionally matching a list of codes provided by command line
-Preprocessing to (re)number approach, transition, departure, and area sections, as well as route declarations under approach, transition, and departure sections.
-Expand certain special declarations inside the source folder and write a file suitable to be read by the game
--Expand airlines declarations with >10 frequency to multiple smaller declarations no greater than 10
--Insert `n` copies of a departure route with *n as the first line
--Replace !<fix> with the lat lon from [airspace] beacons=, works with boundaries=, route=
-Copy the game-ready file to the game's `locations` folder
--Default `locations` folder path can be overridden from an `eatcdev.ini` file

Also added a `.gitignore` to ignore any python cache files created by python, and the `eatcdev.ini` file which would not be suitable to be included in the repo.

### CODE REVIEW

File: tools/.flake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
```
Comment (AdamJCavanaugh) [inline tools/.flake8:1]: I'd probably prefer we resolve the issues to stay inside PEP8 guidelines. I can work on this separately/later, if so desired, though

Comment (ckwng) [reply tools/.flake8:1]: The two ignores are for line too long, which I feel is overly restrictive, and under indent, which exacerbates the line too long. This especially is a problem when full word function names are used for clarity, putting line breaks and parentheses all over the place. If you are able to reduce to 79 line length while maintaining readability though, go ahead, I'm not opposed to it. I just don't know how to get there without making an unreadable mess.

**Comment by AdamJCavanaugh** [comment]
@ckwng I saw you commented in the Discord that you attached a sample eatcdev.ini.example but I don't see it there or here?

Overall, it all looks really good and I'm excited to see more users start using it for more high-quality maps!

**Comment by ckwng** [comment]
> 
> 
> @ckwng I saw you commented in the Discord that you attached a sample eatcdev.ini.example but I don't see it there or here?
> 
> Overall, it all looks really good and I'm excited to see more users start using it for more high-quality maps!

Ah yes, the classic committed but forgot to push....now it is there.

**Review by AdamJCavanaugh — APPROVED** [review]

**Pull request merged**